src/reactionsystem.jl

```diff
@@ -60,9 +60,9 @@ end
 
 function Base.setindex!(rc::ReactionComplex, t::ReactionComplexElement, i...)
     (setindex!(rc.speciesids, t.speciesid, i...);
-     setindex!(rc.speciesstoichs,
-               t.speciesstoich, i...);
-     rc)
+    setindex!(rc.speciesstoichs,
+        t.speciesstoich, i...);
+    rc)
 end
```

Comment:
I think this function should probably just be rewritten as
```julia
function Base.setindex!(rc::ReactionComplex, t::ReactionComplexElement, i...)
    setindex!(rc.speciesids, t.speciesid, i...)
    setindex!(rc.speciesstoichs, t.speciesstoich, i...)
    rc
end
```
I am not sure what is going on with this code and why it is in this style actually...
```

Reply:
incorporated this change